pkg/tecdsa/signing/message.go

```diff
@@ -111,3 +111,23 @@ func (trfm *tssRoundFourMessage) SenderID() group.MemberIndex {
 func (trfm *tssRoundFourMessage) Type() string {
 	return messageTypePrefix + "tss_round_four_message"
 }
+
+// tssRoundFiveMessage is a message payload that carries the sender's
+// TSS round five components.
+type tssRoundFiveMessage struct {
+	senderID group.MemberIndex
+
+	payload   []byte
+	sessionID string
+}
+
+// SenderID returns protocol-level identifier of the message sender.
+func (trfm *tssRoundFiveMessage) SenderID() group.MemberIndex {
+	return trfm.senderID
+}
```

Comment:
I just realized - why do we declare `SenderID()` at all? `SenderID()` is used only from the same package.

I think we used to have some sender-ID-based filtering code in the past, but I think it's gone now.

Reply:
Yeah but now we use it from within `deduplicateBySender` function so we need that getter.